### PR TITLE
Analytics Fixes

### PR DESCRIPTION
**Pull request by ungarson**

### CODE REVIEW

**Comment by ungarson** [comment]
[Add Search Category logging](https://trello.com/c/nMTffM5r/1352-add-search-category-logging)

**Comment by ungarson** [comment]
[Shortpath: send sc=contract | sc=operation instead of suggest](https://trello.com/c/esHQkSCk/1368-shortpath-send-sccontract-scoperation-instead-of-suggest)

File: src/views/extended_search/ResultItem.vue [hunk 1]
```diff
@@ -39,26 +39,26 @@
...
               <router-link class="serp-link" target="_blank"
-                :to="`/${item.body.network}/big_map/${item.body.ptr}/${item.body.key_hash}`">
+                :to="sendTo(`/${item.body.network}/big_map/${item.body.ptr}/${item.body.key_hash}`)">
                 <span class="hash" v-html="highlight(item.body.key)"></span>
               </router-link>
             </template>
              <template v-else-if="item.type === 'token_metadata'">
               <span class="hash">Tokens</span>
               <span class="text--secondary" style="font-size: 20px;"> → </span>
-              <router-link class="serp-link" target="_blank" :to="`/${item.body.network}/${item.value}/tokens/?token_id=${item.body.token_id}`">
```
Comment (m-kus) [inline src/views/extended_search/ResultItem.vue:54]: Shouldn't be remove, no?

File: src/components/SearchBox.vue [hunk 2]
```diff
@@ -359,9 +364,6 @@ export default {
               if (res && res.items) {
                 this.suggests.push(...res.items);
               }
-              if (this.$gtag) {
-                this.$gtag.pageview(`/suggest?text=${text}&sc=${SEARCH_TABS[6]}`);
```
Comment (m-kus) [inline src/components/SearchBox.vue:363]: This was correct, `sc` has to be sent to the search page view

File: src/views/extended_search/ResultItem.vue (hunk 1 above)
Comment (ungarson) [reply src/views/extended_search/ResultItem.vue:54]: @m-kus thanks, I guess, it's the aftermath of merge conflict, fixed

**Comment by ungarson** [comment]
[sc=notfound не трекается аналитикой](https://trello.com/c/Ykwz8deZ/1374-scnotfound-%D0%BD%D0%B5-%D1%82%D1%80%D0%B5%D0%BA%D0%B0%D0%B5%D1%82%D1%81%D1%8F-%D0%B0%D0%BD%D0%B0%D0%BB%D0%B8%D1%82%D0%B8%D0%BA%D0%BE%D0%B9)

File: src/components/SearchBox.vue [hunk 3]
```diff
@@ -253,9 +255,9 @@ export default {
 
       addHistoryItem(this.buildHistoryItem(this.model, value || this.searchText));
       if (this.isModelsArrayInclude("operation") && checkOperation(value)) {
-        this.pushTo(`/${network}/opg/${value}`);
+        this.pushTo(`/${network}/opg/${value}`, 2);
```
Comment (m-kus) [inline src/components/SearchBox.vue:258]: `pushTo(path)` no longer has second arg right?

Comment (ungarson) [reply src/components/SearchBox.vue]: @m-kus thnx fixed

**Pull request merged**